### ISSUE TITLE
Questions about SimpleDisplay

### ISSUE THREAD

**Issue by FinleyPan** [stevenlovegrove/Pangolin#277]
Hello! I recently begin to employ this lib and I am stuck in SimpleDisplay of examples. Everything like  compiling this example and running the building executable goes well except that nothing happened after  clicking button "save_window" or "save_cube", I cannot find where the stored image placed. 
`if( pangolin::Pushed(save_window) )
        pangolin::SaveWindowOnRender("window");`
  I identify above code as saving present window in filename "window", but without suffix(jpg, png, etc.). So whether there exists some default suffix? And whether default storing path is "../build/"(my executable lies in build)

**Comment by stevenlovegrove** [comment]
Hi - they should end up in the working directory - i.e. where ever you run the executable from.

**Comment by dkoj136** [comment]
@stevenlovegrove  Thank you for your reply! I checked the method's defination:  
`void SaveWindowOnRender(std::string prefix)
{
    context->screen_capture.push(std::pair<std::string,Viewport>(prefix, context->base.v) );
}
`My understanding is that the windows to be saved will be pushed into stack when I call method `SaveWindowOnRende(std::string prefix)`, so the screenshots of windows are actually not saved into  my working directory but still in stack? for this reason, I need libpng to pop the screenshots out from stack?
  Forgive me for my weak programming skills. My platform is Ubuntu 16.04 LTS and my compiler is g++， I redownloaded and installed pangolin yesterday, so I am certain that the bug you said has been fixed for me

**Comment by dkoj136** [comment]
Hello! I have installed libpng-dev in my platform(Ubuntu 16.04 LTS)， But when I clicked button "Save_window", still nothing happened. So whether some lib files about libpng need to be placed in my working directory so that screenshot saving can work?

**Comment by dkoj136** [comment]
It worked! Just when I reinstalled pangolin after installing libpng-dev. Thank you! 
  Now a new problem struck me that how to modify the source code if I want to save screenshots into different format(e.g. jpg, tif, gif, bmp, etc.)? Appending suffix directly fails:
  ` pangolin::SaveWindowOnRender("window.jpg");`
  Besides, when I clicked button "Record_Cube", the program exited and compiler notice me:
` terminate called after throwing an instance of 'pangolin::VideoException'  what():  No known video handler for URI 'ffmpeg'
`
 So whether it means a lack of dependency lib like libpng?
  So sorry to trouble you for my trivial questions. Thank you!
